ui/templates/base.html

```diff
@@ -29,6 +29,9 @@
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
     ga('create', '{{ GA_TRACKING_ID }}', 'auto');
+    if('{{ tracking_id }}'){
+      ga('create', '{{ tracking_id }}', 'auto');
+    }
     ga('send', 'pageview');
```

Comment:
It seems like we should also do this for the other tracker after we create it, in the if block above